packages/ndla-ui/src/Frontpage/FrontpageAllSubjects.tsx

```diff
@@ -191,6 +192,23 @@ const FrontpageAllSubjects = ({
   const data: any = [];
   const { t } = useTranslation();
 
+  if (categories.flatMap((c) => c.subjects).length === 0) {
+    return (
+      <StyledWrapper>
+        <ContentLoader width={976} height={200}>
+          <rect x="0" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-1" />
+          <rect x="110" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-2" />
+          <rect x="220" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-3" />
+          <rect x="330" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-4" />
+
+          <rect x="0" y="70" rx="3" ry="3" width="280" height="400" key="rect-2-1" />
+          <rect x="300" y="70" rx="3" ry="3" width="280" height="400" key="rect-2-2" />
+          <rect x="600" y="70" rx="3" ry="3" width="280" height="400" key="rect-2-3" />
+        </ContentLoader>
```

Comment:
Innholdet i ContentLoader vokste utenfor boksen i bunnen og fylte ikke hele bredden. Justerte litt på størrelsene og ser bra ut for meg med denne endringen.
```suggestion
        <ContentLoader width={880} height={270}>
          <rect x="0" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-1" />
          <rect x="110" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-2" />
          <rect x="220" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-3" />
          <rect x="330" y="10" rx="3" ry="3" width="90" height="35" key="rect-1-4" />

          <rect x="0" y="70" rx="3" ry="3" width="280" height="200" key="rect-2-1" />
          <rect x="300" y="70" rx="3" ry="3" width="280" height="200" key="rect-2-2" />
          <rect x="600" y="70" rx="3" ry="3" width="280" height="200" key="rect-2-3" />
        </ContentLoader>
```